en/publish/publish-bytedance.md

```diff
@@ -0,0 +1,122 @@
+
+- **Initial Scene Subpackage**
+
+  This item is optional. <br>
+  When checked, the first scene and its related dependencies will be built into the built-in Asset Bundle under the release package directory assets –– [start-scene](../asset-manager/bundle.md#the-built-in-asset-bundle) to improve the speed of loading assets for the initial scene. The details can be found in the document [Asset Loading for Initial Scenes](publish-wechatgame.md#speed-up-the-loading-of-the-start-scene).
+
+- **Resource Server Address**
+
+  This item is optional and is used to fill in the address of the remote server where the resource is stored. Developers need to manually upload the remote folder in the release package directory to the filled resource server address after the build.
+
+- **Code Directory for Open Data Domains**
+
+  This item is optional and is used to access the open data domain of ByteDance , please refer to [Access the Open Data Domain of ByteDance Mini Game](publish-bytedance-sub-domain.md) for the specific publishing process.
+
+### Run Preview
+
+- Click the **Open** button behind **Build Path** after the build is finished, you can see that the ByteDance mini game project folder **bytedance** is generated under the build directory of the publish package, which already contains the configuration files of bytedance  environment: `game.json` and `project. config.json`.
+
+  ![](./publish-bytedancegame/package.png)
+
+- Use the **ByteDance developer tool** to open the **bytedance** folder generated by the build, you can open the ByteDance game project and preview the debugging game content. Please refer to [Introduction to ByteDance Developer Tools](https://microapp.bytedance.com/docs/zh-CN/mini-game/develop/developer-instrument/development-) for details on how to use the developer tools. assistance/mini-app-developer-instrument).
+
+  ![](./publish-bytedancegame/preview.png)
+
+## Resource Management for ByteDance Mini Game Environment
+
+ByteDance mini games are similar to WeChat mini games in that there is a package size limit, and additional assets over 4MB must be downloaded via network requests.
+
+Cocos Creator has already done the downloading, caching and versioning of remote assets for developers. The specific implementation logic and operation steps are similar to those of WeChat mini games, please refer to [WeChat Mini Game Resource Management](./publish-wechatgame.md#resource-management-for-the-wechat-mini-games).
+
+## Subpackage Loading
+
+Cocos Creator supports ByteDance mini game subpackage function since **v2.4.4**, and the usage is similar to WeChat mini game. Please refer to [Mini Game Subpackage](./subpackage.md).
+
+ByteDance  needs specific version to support subpackage function. The version requirements of ByteDance product are as follows:
```

Comment:
ByteDance  needs specific version to support subpackage function. The version requirements of ByteDance product are as follows:
-->
ByteDance needs specific versions to support the subpackage functionality. The following table shows these requirements: